application/wso2verify/app/src/main/java/org/wso2/carbon/identity/mobile/wso2verify/MyFirebaseMessagingService.kt

```diff
@@ -0,0 +1,46 @@
+package org.wso2.carbon.identity.mobile.wso2verify
+
+import android.util.Log
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import android.content.Intent
+
+/**
+ * Class which handles the Firebase messaging service.
+ */
+class MyFirebaseMessagingService : FirebaseMessagingService() {
+
+    val TAG = "FirebaseMessagingService"
+
+    override fun onMessageReceived(remoteMessage: RemoteMessage){
+
+        Log.d(TAG, "remoteMessage: ${remoteMessage.from}")
+        Log.e(TAG, "From: " + remoteMessage.data.toString())
```

Comment:
is this error level? use debug level and merge with the previous

Reply:
Fixed